codegen_java/project/Publishing.scala

```diff
@@ -0,0 +1,41 @@
+import sbt.Keys._
+import sbt._
+import Artifactory._
+
+ /**
+  * NOTE: This was lifted wholesale from Cromwell.
```

Comment:
It was lifted from Cromwell for use in Rawls, but now that it's been re-lifted into Cromwell from Rawls this comment is out of date 😄 

Any chance we could DRY this out by re-using the original Cromwell edition?

Reply:
I took a look and Cromwell's Publishing does a ton more than this does.  My guess is this was lifted a long time ago.  I'll remove the comment though so it's not confusing.